dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/BvalManagedObjectBuilderImpl.java

```diff
@@ -1,10 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2018 IBM Corporation and others.
+ * Copyright (c) 2017, 2018, 2024 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2017, 2024 IBM Corporation and others.
```
Copyright years are just first and latest update so 2018 should be removed